CONTRIBUTING.md

```diff
@@ -0,0 +1,129 @@
+# 🧑‍💻 Contributing to ZenML Projects
+
+A big welcome and thank you for considering contributing to ZenML Projects! It’s people
+like you that make it a reality for users
```

Comment:
make what a reality?

Alt: maybe something like 'It's people like you who make ZenML Projects what it is!'